packages/antd/src/routing/NavMenuBarAnt.tsx

```diff
@@ -80,7 +80,7 @@ export class NavMenuBarAnt<DataType> extends React.Component<NavMenuProps<DataTy
         states: LocationDependentStateSpaceHandler<UIFragment, UIFragmentSpec, DataType>,
         state: SubStateInContext<UIFragment, UIFragmentSpec, DataType>
     ) {
-        const { label, key, subStates, flat, menuKey, linkStyle } = state;
+        const { label, key, subStates, flat, menuKey, linkStyle, itemClassName } = state;
```

Comment:
After I merged the PR, I saw that typescript was complaining about the unused variable `itemClassName`... I removed it in a fix on master.